### ISSUE TITLE
Not an issue

### ISSUE THREAD

**Issue by ZataraTMM** [DinoDevs/Everwing-Hacks#33]
Is this still beeing updated? as i see, the popular "thomas" hack guy for everwing doesn't upload shit anymore and people keep saying "no hacks work now" but yours seems to.

**Comment by GramThanos** [comment]
This project's aim is not to provide hacks, it is just a proof of how easy it is just to open the console and run a couple of commands in a supposed to be "state-of-the-art" game. That is why the hacks provided was limited to items/reward/resources and not damage or scores, so that it does not affect players that played fair.

Some codes were updated, some of them were removed, some of them needs rework.

So, basically my opinion:
Is the game still hackable? Probably yes.
Why other people does not upload hacks? I don't know, maybe they lack the skills to craft a nice code.

**Comment by ZataraTMM** [comment]
> This project's aim is not to provide hacks, it is just a proof of how easy it is just to open the console and run a couple of commands in a supposed to be "state-of-the-art" game. That is why the hacks provided was limited to items/reward/resources and not damage or scores, so that it does not affect players that played fair.
> 
> Some codes were updated, some of them were removed, some of them needs rework.
> 
> So, basically my opinion:
> Is the game still hackable? Probably yes.
> Why other people does not upload hacks? I don't know, maybe they lack the skills to craft a nice code.

You seem to have the skill. Why not update it? I understand the project and everything as i've loved it and have always been using it and i get your intentions. I don't need any other "damage or score" hacks. I just need the general stuff. not trying to be demandfull or disrespectfull towards you i'm really greatfull for you to create this as it's fun doing the scores. I play the game by myself anyway. That's why i asked if you could update it atleast once? Like, thropies aren't in the game anymore, etc. I'm really thankfull for your time to reply to my post and the time you put into making the project.

**Comment by GramThanos** [comment]
This can be closed.
